src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,114 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String genre;
+    @NotNull
+    private String author;
+    @NotNull
+    private String image;
+    @NotNull
+    private String title;
+    @NotNull
+    private String subtitle;
+    @NotNull
+    private String publisher;
+    @NotNull
+    private String year;
+    @NotNull
+    private int pages;
+    @NotNull
+    private String isbn;
```

Comment:
In these cases where the fields need to be unique ,I would not just add @Column (nullable = false)
I would also add the following:
@Column (nullable = false, unique = true)

```suggestion
    @NotNull
    @Column (nulleable = false, unique = true)
    private String isbn;
```

Reply:
Add the property unique, in the card does not specify that it is unique